Tests/Tests/Core/Extensions/Foundation/CollectionDatedTests.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  CollectionDatedTests.swift
+//  PovioKit_Tests
+//
+//  Created by Yll Fejziu on 14/11/2022.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import XCTest
+import PovioKit
+
+final class CollectionDatedTests: XCTestCase {
+  func test_grouped_returnsEmptyDictionaryOnEmptyArray() {
+    let sut: [Mock] = []
+    XCTAssertTrue(sut.grouped().isEmpty)
```

Comment:
Nit, it is preferable to use `XCTAssertEqual` since in the case of failure the reason why is much easier to infer. We could also leave it as is but with a failure message.